scripts/variantstore/wdl/GvsQuickstartIntegration.wdl

```diff
@@ -34,7 +35,7 @@ workflow GvsQuickstartIntegration {
     File full_wgs_interval_list = "gs://gcp-public-data--broad-references/hg38/v0/wgs_calling_regions.hg38.noCentromeres.noTelomeres.interval_list"
     File full_exome_interval_list = "gs://gcp-public-data--broad-references/hg38/v0/bge_exome_calling_regions.v1.1.interval_list"
     String expected_subdir = if (!chr20_X_Y_only) then "all_chrs/"  else ""
-    File expected_output_prefix = "gs://gvs-internal-quickstart/integration/2024-05-23/" + expected_subdir
+    File expected_output_prefix = "gs://gvs-internal-quickstart/integration/2024-07-03/" + expected_subdir
```

Comment:
updated truth is just an `FT` header line change for most branches, but extensive `OUTSIDE_OF_TARGETS` filtering changes for the exome branch.